dietpi/patch_file

```diff
@@ -601,6 +601,12 @@ _EOF_
 		echo -e "remove me" &> /dev/null
 		#-------------------------------------------------------------------------------
 
+	elif (( $VERSION_CURRENT == 117 )); then
```

Comment:
This patch code should be in the v116 entry above. As the current version of the system will be 116 when the code is run.